xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DeletedDocuments.xml

```diff
@@ -349,7 +390,35 @@ document.observe('xwiki:dom:loaded', function() {
   if ($('documentsTrash')) {
     $('documentsTrash').up('.hidden').removeClassName('hidden');
   }
-});</code>
+});
+
+// we need to wait the complete display of the livetable to get the number of rows
+document.observe('xwiki:livetable:documentsTrash:displayComplete', function() {
+  if ($('documentsTrash')) {
```

Comment:
Is this still needed?

Reply:
Hmmmm I suppose that if the livetable is not loaded then the event cannot be fired. So I guess not.